contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -736,17 +715,26 @@ contract PayloadIGP95 is PayloadIGPMain {
     function action10() internal isActionSkippable(10) {
         address sUSDe_USDC_VAULT = getVaultAddress(7);
         {
-            VaultConfig memory VAULT_sUSDe_USDC = VaultConfig({
-                vault: sUSDe_USDC_VAULT,
-                vaultType: VAULT_TYPE.TYPE_1,
+            // Set supply protocol limits
+            SupplyProtocolConfig memory supplyConfig_ = SupplyProtocolConfig({
```

Comment:
no need to change anything on supply side, only borrow limit

Reply:
again i didnt get then how will we be setting the baseBorrowLimit to 2.5k?